schemas/2015-05-01/microsoft.insights.json

```diff
@@ -0,0 +1,657 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2015-05-01/microsoft.insights.json#",
```

Comment:
>microsoft.insights [](start = 64, length = 18)

please be consistent in names. First letter upper case, so: "Microsoft.Insights".